pytest-selenium/pages/base.py

```diff
@@ -44,3 +44,15 @@ def offscreen_click(self, element):
         # https://stackoverflow.com/a/39918249
         element.send_keys(Keys.ENTER)
         return element
+
+    def offscreen_click_and_wait_for_new_title_to_load(self, element):
+        """Clicks an offscreen element and waits for title to load.
+
+        Clicks the given element, even if it is offscreen, by sending the ENTER key.
+        Returns after loading the last element (title) of the page).
+        """
+        title_before_click = self.title_before_click
```

Comment:
this function is using other functions defined in the subclass, they need to be moved here

Reply:
@Malar-Natarajan this still needs to be fixed

Reply:
@TomWoodward did you mean that I commented the `offscreen_click()` subclass but forgot to remove it or did you mean some other changes?

Reply:
Updated in the latest commit